core/controllers/feedback.py

```diff
@@ -36,6 +36,12 @@ def get(self, exploration_id):
         self.values.update({
             'threads': [t.to_dict() for t in feedback_services.get_all_threads(
```

Comment:
> It gets both feedback threads and suggestion threads. So I left the name as it is.

Then shouldn't this be updated accordingly to feedback_thread_dicts?

Reply:
Yeah, will do that too!